crt_portal/cts_forms/models.py

```diff
@@ -438,6 +446,12 @@ def available_report_fields(self, report):
                 'date_of_intake': format_date(report.create_date, format='long', locale='es_ES'),
                 'outgoing_date': format_date(today, locale='es_ES'),
                 'section_name': section_choices_es.get(report.assigned_section, "no section"),
+            },
+            'zh_hant': {
+                'addressee': report.addressee_zh_hant,
+                'date_of_intake': format_date(report.create_date, format='long', locale='zh_hant'),
+                'outgoing_date': format_date(today, locale='zh_hant'),
```

Comment:
I don't think it's something we need to fix immediately but it's worth refactoring this in the future. As implemented we're calling the entirety of `available_report_fields` for each FormResponseTemplate option rendered on the report details apge.